src/main/java/javax/validation/Configuration.java

```diff
@@ -254,6 +269,17 @@
 	ClockProvider getDefaultClockProvider();
 
 	/**
+	 * Returns the default {@link ValueExtractor} implementations as per the
+	 * specification.
+	 *
+	 * @return the default {@code ValueExtractor} implementations compliant
+	 * with the specification
+	 *
+	 * @since 2.0
+	 */
+	Set<ValueExtractor<?>> getDefaultValueExtractors();
```

Comment:
Instead returning a map we rather could provide a "finder" returning the matching extractor:

    ValueExtractor<?> getDefaultExtractorFor(Type, int)

Leaving it out and adding it in the RI seems good to me.